tetrateci/upload_upstream_release.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+echo "Creating a temporary directory to download $TAG release assets"
+mkdir release-tmp
+cd release-tmp
+
+echo "Fetching the download urls for the $TAG release"
+urls=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/istio/istio/releases/tags/$TAG | jq -r '.assets[] | .browser_download_url')
+
+for url in $urls; do
+    echo "Downloading from $url"
+    wget $url
+done
+
+PACKAGES=$(ls | grep "istio")
+
+for package in $PACKAGES; do
+    echo "Publishing $package"
+    rm -f /tmp/curl.out
+    curl -T ./$package -u$BINTRAY_USER:$API_KEY $BINTRAY_API/$TAG/$package -o /tmp/curl.out
```

Comment:
We need to rename the file as if it is a istio flavor. That is ${TAG} to change to ${TAG}-istio-v0 in the file name.

For example, if the archive name is istio-1.8.3-linux-arm64.tar.gz, then rename it to istio-1.8.3-istio-v0-linux-arm64.tar.gz